### PR TITLE
Simplification of the Databricks Integration tutorial 

### LINKED ISSUE #2355: Integration simplification: Databricks
**Request type:**

- [x] **Addition to an existing guide**: Add the `area-learn` and `enhancement` labels to this issue.

**Use case or feature that should be covered:**

Short tutorial showing how to use Airflow with Databricks. As discussed today. "Easy setup first" structure, with the DatabricksWorkflowTaskGroup and DatabricksNotebookOperator front and center. No SDK or dynamic task mapping.

**Who the guide is intended for:**
Users who want to know how to use Airflow with AWS Redshift.

**Are you the right person to review a draft of the guide for technical accuracy? If not, who is?**
@jlaneve

### PR DESCRIPTION
**Pull request by TJaniF**
Resolves #2355 

Similar comments to the dbt simplifcation PR. The current tutorial is saved and will move to the Examples section :)

Still unsure if I like the Quickstart section up front showing the basic implementation structure. Also again I expect the connection step can be replaced soon :)

### CODE REVIEW

File: learn/airflow-databricks.md [hunk 1]
```diff
@@ -14,14 +14,36 @@ import databricks_tutorial_dag from '!!raw-loader!../code-samples/dags/airflow-d
...
-This tutorial shows how to use the Astro Databricks provider with an example use case analyzing renewable energy data. It also discusses why you would want to [use Databricks with Airflow](#why-use-airflow-with-databricks) and [Alternative ways to run Databricks with Airflow](#alternative-ways-to-run-databricks-with-airflow), if the Astro Databricks Provider doesn't fit your use case.
-
-:::info
-
-All code in this tutorial can be found on [the Astronomer Registry](https://registry.astronomer.io/dags/renewable_analysis_dag/versions/latest).
-
-:::
+This tutorial shows how to use the Astro Databricks provider to run two Databricks notebooks as a Databricks Workflow. It also discusses why you would want to [use Databricks with Airflow](#why-use-airflow-with-databricks) and [Alternative ways to run Databricks with Airflow](#alternative-ways-to-run-databricks-with-airflow), if the Astro Databricks Provider doesn't fit your use case.
+
+## Quickstart
```
Comment (kentdanas) [inline learn/airflow-databricks.md:19]: Same comment here as with the dbt tutorial - I don't want to call this "quickstart", but good to integrate this code snippet in with the intro while linking to the quickstart in the docs. https://github.com/astronomer/astro-provider-databricks/blob/main/quickstart/astro-cli.md

File: learn/airflow-databricks.md [hunk 2]
```diff
@@ -14,14 +14,36 @@ import databricks_tutorial_dag from '!!raw-loader!../code-samples/dags/airflow-d
...
 
-
-This tutorial shows how to use the Astro Databricks provider with an example use case analyzing renewable energy data. It also discusses why you would want to [use Databricks with Airflow](#why-use-airflow-with-databricks) and [Alternative ways to run Databricks with Airflow](#alternative-ways-to-run-databricks-with-airflow), if the Astro Databricks Provider doesn't fit your use case.
-
-:::info
-
-All code in this tutorial can be found on [the Astronomer Registry](https://registry.astronomer.io/dags/renewable_analysis_dag/versions/latest).
-
-:::
+This tutorial shows how to use the Astro Databricks provider to run two Databricks notebooks as a Databricks Workflow. It also discusses why you would want to [use Databricks with Airflow](#why-use-airflow-with-databricks) and [Alternative ways to run Databricks with Airflow](#alternative-ways-to-run-databricks-with-airflow), if the Astro Databricks Provider doesn't fit your use case.
```
Comment (kentdanas) [inline learn/airflow-databricks.md:17]:
```suggestion
This tutorial shows how to use the Astro Databricks provider to run two Databricks notebooks as a Databricks Workflow. If you use Databricks services other than Databricks Workflows, see [Alternative ways to run Databricks with Airflow](#alternative-ways-to-run-databricks-with-airflow).
```

File: learn/airflow-databricks.md [hunk 3]
```diff
@@ -49,224 +71,44 @@ To get the most out of this tutorial, make sure you have an understanding of:
...
+- The [Astro CLI](https://docs.astronomer.io/astro/cli/overview) with an empty Astro project created by `astro dev init`.
 - Access to a Databricks workspace. See [Databricks' documentation](https://docs.databricks.com/getting-started/index.html) for instructions. You can use any workspace that has access to the [Databricks Workflows](https://docs.databricks.com/workflows/index.html) feature. You need a user account with permissions to create notebooks and Databricks jobs. You can use any underlying cloud service, and a [14-day free trial](https://www.databricks.com/try-databricks) is available.
-- Access to an [object storage supported by the Astro Python SDK](https://astro-sdk-python.readthedocs.io/en/stable/supported_file.html). This tutorial uses an [AWS S3](https://aws.amazon.com/s3/) bucket.
-- Access to a [relational database supported by the Astro Python SDK](https://astro-sdk-python.readthedocs.io/en/stable/supported_databases.html). This tutorial uses [PostgreSQL](https://www.postgresql.org/).
 
 ## Step 1: Configure your Astro project
 
-An Astro project contains all of the files you need to run Airflow locally.
```
Comment (kentdanas) [inline learn/airflow-databricks.md:59]: Same comment as with dbt - you can leave this in. We still want the context for how to start a project :)

File: learn/airflow-databricks.md [hunk 4]
```diff
@@ -49,224 +71,44 @@ To get the most out of this tutorial, make sure you have an understanding of:
...
     ```text
-    AIRFLOW__CORE__ALLOWED_DESERIALIZATION_CLASSES = airflow\.* astro\.* astro_databricks\.*
+    AIRFLOW__CORE__ALLOWED_DESERIALIZATION_CLASSES = airflow\.* astro_databricks\.*
     ```
 
-## Step 2: Prepare the data
+## Step 2: Create Databricks Notebooks
 
-In this tutorial, you analyze two renewable energy datasets to calculate the percentage of electricity coming from solar, wind, and hydro power for a given country over time. In this step, you prepare your data for later analysis.
+You can orchestrate any existing Databricks notebooks in a Databricks Workflow using the Astro Databricks provider. If you don't have Databricks notebooks ready follow the steps below to create two notebooks to use in this tutorial.
```
Comment (kentdanas) [inline learn/airflow-databricks.md:93]:
```suggestion
You can orchestrate any existing Databricks notebooks in a Databricks Workflow using the Astro Databricks provider. If you don't have Databricks notebooks ready, follow the steps below to create two notebooks to use in this tutorial.
```

File: learn/airflow-databricks.md [hunk 5]
```diff
@@ -280,67 +122,29 @@ To analyze your data, you need to use three data tools external to Airflow: Data
...
-
-:::
-
-## Step 6: Create your DAG
-
-
-1. In your `dags` folder, create a file called `renewable_analysis_dag.py`.
-
-2. Copy and paste the following DAG code into the file:
+2. Copy and paste the following DAG code into the file. Make sure to replace the `DATABRICKS_LOGIN_EMAIL` variable with your Databricks login email. If you are using your existing Databricks notebooks you also will have to adjust the notebook names and potentially have to add additional tasks defined with the DatabricksNotebookOperator to orchestrate more than two Databricks notebooks in this Databricks Workflow.
```
Comment (kentdanas) [inline learn/airflow-databricks.md:129]:
```suggestion
2. Copy and paste the following DAG code into the file. Replace the `DATABRICKS_LOGIN_EMAIL` variable with your Databricks login email. If you already had Databricks notebooks and did not create new ones in Step 2, replace the notebook names in the DatabricksNotebookOperators.
```

**Review by kentdanas — APPROVED** [review]
Looks great! Just a few minor edits.

File: learn/airflow-databricks.md (hunk 3 above)
Comment (TJaniF) [reply learn/airflow-databricks.md:59]: 😅 I might have gotten ahead of myself a little bit when deleting things here.

**Review by TJaniF — COMMENTED** [review]
Thank you @kentdanas ! :) 

@lzdanski and @jwitz the second tutorial simplification ready for you :) 

@jlaneve cc for you if you want to have a look. I took out as much as I could of the not directly Astro Databricks provider relevant code :)

**Review by jwitz — APPROVED** [review]
LGTM!

**Review by TJaniF — COMMENTED** [review]
Thank you for the review @jwitz ! I accepted everything and will squashmerge :)

**Pull request merged**